src/main/webapp/assets/modules/common/confirmOnExitDirective.js

```diff
@@ -0,0 +1,27 @@
+techlooper.directive('confirmOnExit', function ($rootScope) {
+  return {
+    require: '^form',
+    restrict: 'A',
+    link: function (scope, element, attrs, ngModelCtrl) {
```

Comment:
'ngModelCtrl' is defined but never used.<br>'attrs' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.